app/globals.css

```
@@ -41,4 +41,56 @@
+    color: var(--shiki-light) !important;
+  }
+
+  .dark .blog-prose pre.shiki,
+  .dark .blog-prose pre.shiki span {
+    color: var(--shiki-dark) !important;
+    background-color: var(--shiki-dark-bg) !important;
+  }
+
+  .blog-prose pre code {
+    @apply !border-0 !bg-transparent !p-0 !text-sm;
+    color: inherit !important;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)